config/initializers/opentracing.rb

```diff
@@ -0,0 +1,23 @@
+# noinspection RubyResolve
+OpenTracing.global_tracer = Jaeger::Client.build(
+  service_name: Figaro.env.JAEGER_SERVICE_NAME,
+  reporter: Jaeger::Reporters::RemoteReporter.new(
+    sender: Jaeger::HttpSender.new(
```

Comment:
can we use UDP protocol? so it will not block if network error from tracing agent/collector?